package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "serverless-openapi-typescript",
-  "version": "1.0.10",
+  "version": "2.0.0",
```

Comment:
I changed major version because we broke the API.
operationId value will be different after this version